client/lib/abtest/active-tests.js

```diff
@@ -134,4 +134,11 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	privateByDefault: {
```

Comment:
The test definitation is missing `defaultVariation` which is required. Without it, it hangs when creating the site and there's an error in the console saying `defaultVariation` is missing.

Reply:
Pushed fix